olive/systems/python_environment/python_environment_system.py

```diff
@@ -76,7 +90,48 @@ def run_pass(
         """
         Run the pass on the model at a specific point in the search space.
         """
-        raise ValueError("PythonEnvironmentSystem does not support running passes.")
+        import importlib
+
+        try:
+            importlib.import_module("olive")
+        except ImportError:
+            raise ValueError("PythonEnvironmentSystem does not support running passes without olive installed.")
+
```

Comment:
dead code since the python env system always have olvie installed.

Reply:
If it is not olive_managed_env, user may not have olive installed in their prepared environment.